draft-ietf-httpbis-digest-headers.md

```diff
@@ -1102,6 +1102,18 @@ or by sending multiple representation-data-digest values from which the receiver
 Endpoints are advised that sending multiple values consumes resources,
 which may be wasted if the receiver ignores them (see {{digest}}).
 
+## Resource exhaustion
+
+`Digest` validation consumes computational resources.
+To mitigate the risk of resource exhaustion, it is advisable to:
+
+- limit the set of digest-algorithms to the ones that
+  will be effectively used;
```

Comment:
who "limits" what's "used"? Is this a intended to say a receiver should limit the set of algorithms that it will validate for any given Digest field, or are you saying something different?

Reply:
Example: if I just expect to use "sha512", I could configure the validation middleware to ignore other values.